src/actor_configs.js

```diff
@@ -0,0 +1,302 @@
+     * @memberof ApifyClient.actorConfigs
+     * @instance
+     * @param {Object} options
+     * @param options.token
+     * @param {Number} [options.offset=0] - Number of array elements that should be skipped at the start.
+     * @param {Number} [options.limit=1000] - Maximum number of array elements to return.
+     * @param {Number} [options.desc] - If 1 then the objects are sorted by the createdAt field in descending order.
+     * @param callback
+     * @returns {PaginationList}
+     */
+    listActorConfigs: (requestPromise, options) => {
+        const { baseUrl, token, offset, limit, desc } = options;
+
+        checkParamOrThrow(baseUrl, 'baseUrl', 'String');
+        checkParamOrThrow(token, 'token', 'String');
+        checkParamOrThrow(limit, 'limit', 'Maybe Number');
+        checkParamOrThrow(offset, 'offset', 'Maybe Number');
+        checkParamOrThrow(desc, 'desc', 'Maybe Boolean');
+
+        const query = { token };
+
+        if (limit) query.limit = limit;
+        if (offset) query.offset = offset;
+        if (desc) query.desc = 1;
+
+        return requestPromise({
+            url: `${baseUrl}${BASE_PATH}`,
+            json: true,
+            method: 'GET',
+            qs: query,
+        })
+            .then(pluckData)
+            .then(parseDateFields);
+    },
+
+    /**
+     * Creates a new actor config.
+     *
+     * @memberof ApifyClient.actorConfigs
+     * @instance
+     * @param {Object} options
+     * @param options.token
+     * @param {Object} options.actorConfig Object containing configuration of the actor config
+     * @param callback
+     * @returns {ActorConfig}
+     */
+    createActorConfig: (requestPromise, options) => {
+        const { baseUrl, token, actorConfig } = options;
+
+        checkParamOrThrow(baseUrl, 'baseUrl', 'String');
+        checkParamOrThrow(token, 'token', 'String');
+        checkParamOrThrow(actorConfig, 'actorConfig', 'Object');
+
+        return requestPromise({
+            url: `${baseUrl}${BASE_PATH}`,
+            json: true,
+            method: 'POST',
+            qs: { token },
+            body: actorConfig,
+        })
+            .then(pluckData)
+            .then(parseDateFields);
+    },
+
+    /**
+     * Updates actorConfig.
+     *
+     * @memberof ApifyClient.actorConfig
+     * @instance
+     * @param {Object} options
+     * @param options.token
+     * @param {String} options.actorConfigId - Unique actorConfig ID
+     * @param {Object} options.actorConfig
+     * @param callback
+     * @returns {ActorConfig}
+     */
+    updateActorConfig: (requestPromise, options) => {
+        const { baseUrl, token, actorConfigId, actorConfig } = options;
+
+        checkParamOrThrow(baseUrl, 'baseUrl', 'String');
+        checkParamOrThrow(token, 'token', 'String');
+
+        if (actorConfigId) checkParamOrThrow(actorConfigId, 'actorConfigId', 'String');
```

Comment:
Since you require `actorConfig` to be an object, you can move  ` checkParamOrThrow(actorConfig, 'actorConfig', 'Object');` to the top and then simplify:

```
if (actorConfigId) checkParamOrThrow(actorConfigId, 'actorConfigId', 'String');
else if (actorConfig) checkParamOrThrow(actorConfig.id, 'actorConfig.id', 'String');
else checkParamOrThrow(actorConfigId, 'actorConfigId', 'String');
```

to

```
if (actorConfigId) checkParamOrThrow(actorConfigId, 'actorConfigId', 'String');
else checkParamOrThrow(actorConfig.id, 'actorConfig.id', 'String');
```

or  maybe better

```
if (actorConfigId && actorConfigId.id)  checkParamOrThrow(actorConfig.id, 'actorConfig.id', 'String');
else checkParamOrThrow(actorConfigId, 'actorConfigId', 'String');
```